### ISSUE TITLE
Could you please show training loss？

### ISSUE THREAD

**Issue by lightningsoon** [TotallyFine/SSD-pytorch#4]
Thanks for your work.I have successfully run these code.But the train classification loss  retain at up 3.
I would know whether the situation is correct.
![image](https://user-images.githubusercontent.com/20694908/66626373-92f7b680-ec2a-11e9-9190-327d78793514.png)
